### DISCUSSION TITLE
[BUG] 为何github action部署失败，无法看到详细日志

### DISCUSSION THREAD

**Discussion by inoribea** [dreamhunter2333/cloudflare_temp_email#499] [General]
## 复现步骤

![图片](https://github.com/user-attachments/assets/20debf56-4ee2-473e-80c9-9923a64da22b)

## 部署方式

- [ ] cli 部署
- [x]  用户界面部署

## 浏览器环境

已配置backend的toml,复制修改后填入github仓库的变量里

![图片](https://github.com/user-attachments/assets/780f45fd-e9d6-4f87-8371-1aeb6cc45b57)

**Comment by dreamhunter2333** [comment]
因为日志中回出现密码信息，隐藏了日志，将此处直接改为 `pnpm run deploy` 即可，并去掉后面的 if

https://github.com/dreamhunter2333/cloudflare_temp_email/blob/0308f518da225fbcd89d47ee207567489d17df66/.github/workflows/backend_deploy.yaml#L35

**Comment by shulng** [comment]
> 因为日志中回出现密码信息，隐藏了日志，将此处直接改为 `pnpm run deploy` 即可，并去掉后面的 if
> 
> https://github.com/dreamhunter2333/cloudflare_temp_email/blob/0308f518da225fbcd89d47ee207567489d17df66/.github/workflows/backend_deploy.yaml#L35

![Screenshot_20241201_031339.jpg](https://github.com/user-attachments/assets/7288c587-8e7c-43de-8cd9-8de8c3f26cfa)

✘ [ERROR] The `nodejs_compat` compatibility flag cannot be used in conjunction with the legacy `--node-compat` flag. If you want to use the Workers `nodejs_compat` compatibility flag, please remove the `--node-compat` argument from your CLI command or `***` from your config file.
🪵  Logs were written to "/home/runner/.config/.wrangler/logs/wrangler-2024-11-30_19-21-49_594.log"
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.

**Comment by 3377** [comment]
解决了吗，我也同样的错误

**Reply by shulng** [reply]
> 解决了吗，我也同样的错误

解决了

**Comment by 3377** [comment]
@dreamhunter2333 @shulng 
![image](https://github.com/user-attachments/assets/214513cc-2c25-44b4-aed9-06d13873718d)
我的报这个错。

**Reply by shulng** [reply]
> @dreamhunter2333 @shulng ![image](https://private-user-images.githubusercontent.com/15319161/391849288-214513cc-2c25-44b4-aed9-06d13873718d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMyMTAwMTksIm5iZiI6MTczMzIwOTcxOSwicGF0aCI6Ii8xNTMxOTE2MS8zOTE4NDkyODgtMjE0NTEzY2MtMmMyNS00NGI0LWFlZDktMDZkMTM4NzM3MThkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAzVDA3MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmMzg1MDNmODFiOTcwYzY5MDBmMzI4YmIwN2JjYWJiZTdjYWZmNzhkMmJlMWJlZGU3OGQ3ZTdkMTI5NGMyMTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.uiwBOwX220jwcM_Scsi6-14hbRi4rGckO_1GTNoVGIc) 我的报这个错。

把这个注释掉
node_compat = true

**Comment by 3377** [comment]
![image](https://github.com/user-attachments/assets/e7daef68-4a62-403b-b277-59ff461f0f71)
的确，我用ai也解决了，这是个BUG啊，醉了。

**Comment by 3377** [comment]
![image](https://github.com/user-attachments/assets/b030ca4b-67ac-4cf0-a0d1-a2d12f23d42d)
还是有报错。。。
@shulng 你踩过这个坑吗

**Reply by dreamhunter2333** [reply]
这已经部署成功了，你脚本后面的 if 没删掉吧

**Comment by dreamhunter2333** [comment]
示例的toml 中只有 compatibility_flags = [ "nodejs_compat" ]
